velox/core/PlanNode.h

```diff
@@ -1741,6 +1741,15 @@ class OrderByNode : public PlanNode {
         sortingKeys.size(),
         sortingOrders.size(),
         "Number of sorting keys and sorting orders in OrderBy must be the same");
+    std::unordered_set<std::string> sortingKeySet;
+    // Reject duplicated sorting keys.
+    for (int i = 0; i < sortingKeys.size(); i++) {
+      auto filed = sortingKeys.at(i);
+      VELOX_USER_CHECK(filed, "OrderBy sorting key can not be null");
```

Comment:
use VELOX_USER_CHECK_NOT_NULL

Use consistent error messages: "Sorting key cannot be null"